src/components/videoCard/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useWatchLater } from "../../context/watchLater";
+import { ACTION } from "../../actions/watchLater";
+
+const VideoCard = ({ video, handlePlayListModal, isDeleteOption = false }) => {
+  const navigator = useNavigate();
+  const [watchLater, setWatchLater] = useState(false);
+
+  const { dispatchWatchLaterList } = useWatchLater();
+
+  function addToList(video) {
+    dispatchWatchLaterList({
+      type: ACTION.ADD_TO_LIST,
+      payload: {
+        video: video,
+      },
+    });
+    setWatchLater(false);
+  }
+
+  function removeFromList(video) {
+    dispatchWatchLaterList({
+      type: ACTION.REMOVE_FROM_LIST,
+      payload: {
+        video: video,
+      },
+    });
+    setWatchLater(false);
+  }
+
+  return (
+    <div className="position-relative">
+      <div className="video-item">
+      <abbr title="Add to playlist">
+          <i
+            onClick={() => handlePlayListModal(video._id)}
+            className="fa fa-list list-icon"
+            aria-hidden="true"
+          ></i>
+        </abbr>
+        <div onClick={() => navigator(`/video/${video._id}`)}>
+          <img src={video.staticImg} alt={video.title} />
+        
```

Comment:
you can also use link to get images with respect to id 
https://img.youtube.com/vi/<>video id here</>/maxresdefault.jpg

Reply:
okay Thanks bro.